argocd/resource_argocd_application_test.go

```diff
@@ -168,7 +168,7 @@ func TestAccArgoCDApplication_Helm_FileParameters(t *testing.T) {
 			{
 				Config: testAccArgoCDApplicationHelm_FileParameters(acctest.RandomWithPrefix("test-acc")),
 				// Setting up tests for this is non-trivial so it is easier to test for an expected failure (since file does not exist) than to test for success
-				ExpectError: regexp.MustCompile(`(?s)Error: failed parsing.*--set-file data`),
+				ExpectError: regexp.MustCompile(`(?s)Error:\s+failed\s+parsing.*--set-file data`),
```

Comment:
This ensures there could be multiple line spaces between the parts of the error message (as it's currently the case)